transcendence-app/src/oauth42/oauth42.controller.ts

```diff
@@ -1,11 +1,19 @@
 import { Controller, Get, UseGuards } from '@nestjs/common';
+import {
+  ApiBadGatewayResponse,
+  ApiFoundResponse,
+  ApiTags,
+} from '@nestjs/swagger';
```

Comment:
Do we need to add this tags in every controller right?

Reply:
Correct, with the `ApiTags` decorator, we can group routes. If we don't add this decorator, they get added to the `default` tag.

And if the route has more than one response type, we need to add the decorators for them (if we want those response types to appear in SwaggerUI).